environment.yml

```diff
@@ -5,11 +5,47 @@ channels:
   - conda-forge
   - bioconda
   - defaults
+  - hcc
 dependencies:
-  - conda-forge::python=3.7.3
-  - conda-forge::markdown=3.1.1
-  - conda-forge::pymdown-extensions=6.0
-  - conda-forge::pygments=2.5.2
-  # TODO nf-core: Add required software dependencies here
-  - bioconda::fastqc=0.11.8
-  - bioconda::multiqc=1.7
+  ## conda-forge packages
+  - conda-forge::python=3.6.10
+  - conda-forge::markdown=3.2.2
+  - conda-forge::pymdown-extensions=7.1
+  - conda-forge::pygments=2.6.1
+  - conda-forge::pigz=2.3.4
+  - conda-forge::r-base=3.6.2
+  - conda-forge::bc=1.07.1
+
+  ## bioconda packages
+  ## common
+  - bioconda::fastqc=0.11.9
+  - bioconda::parallel-fastq-dump=0.6.6
+  - bioconda::sra-tools=2.10.3
+  - bioconda::fastp=0.20.1
+  - bioconda::samtools=1.9
+  - bioconda::bedtools=2.29.2
+  - bioconda::multiqc=1.8
```

Comment:
```suggestion
  - bioconda::multiqc=1.9
```

Reply:
😆

Reply:
I think I will create an honorary PR for this one 🥇